config/.env.example

```diff
@@ -1,5 +1,15 @@
 DEBUG=True
+# START_FEATURE debug_toolbar
+DEBUG_TOOLBAR=True
+# END_FEATURE debug_toolbar
 SECRET_KEY=[FILL_ME_IN]
 DATABASE_URL=sqlite:///db.sqlite3
 GOOGLE_OAUTH2_KEY=[FILL_ME_IN]
 GOOGLE_OAUTH2_SECRET=[FILL_ME_IN]
+# START_FEATURE sentry
+SENTRY_DSN=[FILL_ME_IN]
+# END_FEATURE sentry
+# START_FEATURE django_react
+# this setting is only relevant to local development
```

Comment:
If this setting is only used in local development, should it have an `if DEBUG` clause?

Reply:
I'm not sure what you mean - this is the example `.env` file. Can .envs have conditional logic?